owa/app/js/components/manage_autogeneration_option/ManageAutoGenerationOption.jsx

```diff
@@ -1,52 +1,33 @@
 import React, { Component } from 'react';
-import { Button, Container, Row, Col, FormGroup, Label, Input, Table,
-    Modal, ModalHeader, ModalBody, ModalFooter, Alert } from 'reactstrap';
+import {
+  Button,
+  Container,
+  Row,
+  Col,
+  FormGroup,
+  Label,
+  Input,
+  Table,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Alert
```

Comment:
is this formatting standard?

Reply:
Yes. It's what I've been doing and I think its eslint acceptable way of doing it. But I'm open to any other suggestion or better way of doing it.